packages/api/src/schema/api-schema.yaml

```
@@ -2138,11 +2544,198 @@ paths:
+        which will work better for users with unreliable or slow network
+        connections. If you want a simpler implementation though, you should
+        just use a direct upload.
+
+
+        ## Direct Upload
+
+        For a direct upload, make a PUT request to the URL received in the url
+        field of the response above, with the raw video file as the request
+        body. response above:
+
+
+        ## Resumable Upload
+
+        Livepeer supports resumable uploads via Tus(opens in a new tab). This
+        section provides a simple example of how to use tus-js-client to upload
+        a video file.
+
+
+        From the previous section, we generated a URL to upload a video file to
+        Livepeer on POST /api/asset/request-upload. You should use the
+        tusEndpoint field of the response to upload the video file and track the
+        progress:
+
+
+        ```javascript 
+
+        # This assumes there is an `input` element of `type="file"` with id
+        `fileInput` in the HTML
+
+
+        const input = document.getElementById('fileInput');
+
+        const file = input.files[0];
+
+        const upload = new tus.Upload(file, {
+          endpoint: tusEndpoint, // URL from `tusEndpoint` field in the
+        `/request-upload` response
+          metadata: {
+            filename,
+            filetype: 'video/mp4',
+          },
+          uploadSize: file.size,
+          onError(err) {
+            console.error('Error uploading file:', err);
+          },
+          onProgress(bytesUploaded, bytesTotal) {
+            const percentage = ((bytesUploaded / bytesTotal) * 100).toFixed(2);
+            console.log('Uploaded ' + percentage + '%');
+          },
+          onSuccess() {
+            console.log('Upload finished:', upload.url);
+          },
+        });
+
+        const previousUploads = await upload.findPreviousUploads();
+
+        if (previousUploads.length > 0) {
+          upload.resumeFromPreviousUpload(previousUploads[0]);
+        }
+
+        upload.start();
+
+        ```
+
+
+        > Note: If you are using tus from node.js, you need to add a custom URL
+        storage to enable resuming from previous uploads. On the browser, this
+        is enabled by default using local storage. In node.js, add urlStorage:
+        new tus.FileUrlStorage("path/to/tmp/file"), to the UploadFile object
+        definition above.
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/request-upload"
+      responses:
+        "200":
+          description: Success
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/upload-response"
+        default:
+          description: Error
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/error"
+  /asset/import:
```

This API is now
```suggestion
  /asset/upload/url:
```
(everything else same, but remember to update the text below that repeats the path)